node_modules/oae-core/activity/activity.html

```diff
@@ -61,7 +61,7 @@
                             <div class="media-body">
                                 <h4 class="media-heading">
                                     {if item.comment.author['oae:profilePath']}
-                                        <a href="${item.comment.author['oae:profilePath']|profilePath}">${item.comment.author.displayName|encodeForHTML}</a>
+                                        <a href="${item.comment.author['oae:profilePath']|profilePath}" aria-hidden="true">${item.comment.author.displayName|encodeForHTML}</a>
```

Comment:
When I try this with VoiceOver enabled, tabbing through the comments still stops at both the thumbnail image and the user link, causing the name of the user to be read twice. Should we also be removing the tabindex from these thumbnail images to ensure that the name is not repeated?